edgelet/hsm-sys/azure-iot-hsm-c/src/edge_pki_openssl.c

```diff
@@ -1496,6 +1613,7 @@ KEY_HANDLE create_cert_key(const char* key_file_name)
     EVP_PKEY* evp_key;
     CERT_KEY *cert_key;
 
+    initialize_openssl();

```

Comment:
Was this being called before?

Reply:
No this was not being done before and this code was never executed prior to this PR. I noticed this during dev.